index.html

```diff
@@ -9,9 +10,113 @@
   <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
   <script src="scripts/main.js"></script>
 </head>
-<body>
-  <div class="modalMobile"></div>
-  <div class="modalDesktop"></div>
+
+<body onload="loadProjects()">
+  <div class="modalMobile dispNone" id="modalMobileID">
+    <div>
+      <img onclick="closeMobileModal()" class="crossPosition" src="resources/ic_cross.svg" alt="A cross">
+      <h4 class="modalProjectNameMobile" id="mobileProject">Project name goes here</h4>
+    </div>
+    <ul class="whiteul whiteulpadding">
+      <li class="whiteli">HTML/CSS</li>
+      <li class="whiteli">Ruby on Rails</li>
+      <li class="whiteli">JavaScript</li>
+    </ul>
+
+    <img class="arrowLeft" src="resources/Vector 4.svg" alt="Arrow left">
+    <img class="arrowRight" src="resources/Vector 5.svg" alt="Arrow right">
+
+    <img class="fullWidth4" id="bigPicMobile" src="resources/Rectangle 21.png" alt="Laptop and tangerine">
+
+    <div class="smallImages">
+      <img id="smallPicMobile1" class="firstSmallImg" src="resources/Rectangle 21.png" alt="Laptop and tangerine">
+      <img id="smallPicMobile2" src="resources/Rectangle 21.png" alt="Laptop and tangerine">
+      <img id="smallPicMobile3" src="resources/Rectangle 21.png" alt="Laptop and tangerine">
+      <img id="smallPicMobile4" src="resources/Rectangle 21.png" alt="Laptop and tangerine">
+    </div>
+
+    <div class="modalTextMobile">
+      <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Natus totam, animi veniam qui vitae ut unde quo in
+        nesciunt nulla quos inventore pariatur deserunt quis laboriosam laudantium nobis possimus deleniti.</p>
+
+      <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cumque aspernatur corporis est pariatur sint ratione
+        dolorum inventore eligendi odit, quibusdam quas maiores nobis quia! Hic repudiandae molestiae unde iusto soluta
+        aut nobis sed dolor iure quibusdam.</p>
+    </div>
+
+    <button onclick="seeLive()" class="whiteButton">
```

Comment:
- [ ] Please make sure to handle any events from only the JavaScript file as this goes against best practices. Behavior (Javascript) and presentation (HTML) should be separated. ✔
- [ ] This applies to all the `onClick` attribute you've used in the "see project" buttons.